ts/parts/SplineSeries.ts

```diff
@@ -253,10 +253,10 @@ seriesType<Highcharts.SplineSeries>(
             // */
             ret = [
                 'C',
-                pick(lastPoint.rightContX, lastPoint.plotX),
-                pick(lastPoint.rightContY, lastPoint.plotY),
-                pick(leftContX, plotX),
-                pick(leftContY, plotY),
+                pick(lastPoint.rightContX, lastPoint.plotX) as any,
+                pick(lastPoint.rightContY, lastPoint.plotY) as any,
+                pick(leftContX, plotX) as any,
+                pick(leftContY, plotY) as any,
```

Comment:
This is just missing a final non-null/undefined value: `pick(…, …, 0)` to avoid the `any` cast.

Reply:
Thanks, will do a bit of rewrite here.